packages/optimizely-sdk/lib/core/decision_service/index.tests.js

```diff
@@ -75,9 +75,11 @@ describe('lib/core/decision_service', function() {
 
       it('should return null if the user does not meet audience conditions', function () {
         assert.isNull(decisionServiceInstance.getVariation('testExperimentWithAudiences', 'user3', {foo: 'bar'}));
-        assert.strictEqual(2, mockLogger.log.callCount);
+        assert.strictEqual(7, mockLogger.log.callCount);
         assert.strictEqual(mockLogger.log.args[0][1], 'PROJECT_CONFIG: User user3 is not in the forced variation map.');
-        assert.strictEqual(mockLogger.log.args[1][1], 'DECISION_SERVICE: User user3 does not meet conditions to be in experiment testExperimentWithAudiences.');
+        assert.strictEqual(mockLogger.log.args[1][1], 'DECISION_SERVICE: Evaluating audiences for experiment "testExperimentWithAudiences": "11154".');
```

Comment:
I would expect this to be something more like
```suggestion
        assert.strictEqual(mockLogger.log.args[1][1], 'DECISION_SERVICE: Evaluating audiences for experiment "testExperimentWithAudiences": ["11154"].');
```
(Same thing in other test cases below)